lib/matplotlib/axes/_axes.py

```diff
@@ -632,22 +632,21 @@ def annotate(self, *args, **kwargs):
         s : string
             label
 
-        xy : (x, y)
-            position of element to annotate
+        xy : (x, y) , default: "data"
```

Comment:
Use as many lines as you need.

Not everything can be documented in about half a tweet

On Tue, Dec 22, 2015, 01:34 Devashish Deshpande <notifications@github.com>
wrote:

> In lib/matplotlib/axes/_axes.py
> <https://github.com/matplotlib/matplotlib/pull/5709#discussion_r48225539>:
>
> > @@ -632,22 +632,21 @@ def annotate(self, *args, **kwargs):
> >          s : string
> >              label
> >
> > -        xy : (x, y)
> > -            position of element to annotate
> > +        xy : (x, y) , default: "data"
>
> Oh sorry I'll remove the default value for xy. Yes what you said makes it
> much clearer but then I think I'll have to use the next line. Is it fine if
> I do so? Ideally I don't really want to do that though...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/5709/files#r48225539>.
>